.env

```
@@ -0,0 +1,32 @@
+MINIO_ACCESS_KEY=
+MINIO_SECRET_KEY=
+MINIO_USE_SSL=false
+MINIO_SINGLE_BUCKET=rp-storage
+
+# S3 configuration
+DATASTORE_REGION=eu-central-1
+S3_ENDPOINT=https://s3.eu-central-1.amazonaws.com
+S3_ACCESS_KEY=
+S3_SECRET_KEY=
+S3_SINGLE_BUCKET=rp-s3-storage
+S3_USE_SSL=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)